src/mol-plugin-state/manager/structure/measurement.ts

```diff
@@ -125,6 +127,8 @@ class StructureMeasurementManager extends StatefulPluginComponent<StructureMeasu
 
         const state = this.plugin.state.data;
         await PluginCommands.State.Update(this.plugin, { state, tree: update, options: { doNotLogTiming: true } });
+        return { distance: distance.selector, distanceRep: distanceRep.selector };
```

Comment:
Can you please either name the first argument to `distanceSelection` and the second to `distanceRepresentation`, or just use `selection` and `representation` everywhere?

Reply:
Sure thing, made it uniform in https://github.com/molstar/molstar/pull/1087/commits/f986688857838effeb2d0a101ca878a54f572a44